src/features/student/components/toggles/TraceAiUsageToggle/TraceAiUsageToggle.vue

```diff
@@ -0,0 +1,30 @@
+<script setup lang="ts">
+import { AvToggle, type AvToggleProps } from '@avenirs-esr/avenirs-dsav'
+import { useAttrs } from 'vue'
+import { useI18n } from 'vue-i18n'
+
+interface TraceAiUsageToggleProps extends Omit<AvToggleProps, 'description' | 'activeText' | 'inactiveText'> {
+  description?: string
+  activeText?: string
+  inactiveText?: string
+}
```

Comment:
```suggestion
interface TraceAiUsageToggleProps extends Omit<AvToggleProps, 'description' | 'activeText' | 'inactiveText'>{
  description?: string
}
```
A priori tu n'utilises jamais ces propriétés dans ton code